<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgGame</class>
 <widget class="QDialog" name="CDlgGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>396</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game Info</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QTreeWidget" name="treeSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is a list of all the settings for your game. 
Double-click on the one you want to edit.</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnDefaults">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore Defaults</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <attribute name="title">
     <string>Sprites</string>
    </attribute>
    <widget class="QToolButton" name="btnAddObject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>25</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Add New Object</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeObjects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is a list of all the objects contained in your game. 
Double-click on the one you want to edit.</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QToolButton" name="btnDeleteObject">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Delete Object</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <attribute name="title">
     <string>Sounds</string>
    </attribute>
    <widget class="QTreeWidget" name="treeSounds">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is a list of all the sounds contained in your game. 
Double-click on the one you want to edit.</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QToolButton" name="btnDeleteSound">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Delete Sound</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAddSound">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>25</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Add New Sound</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <attribute name="title">
     <string>Images</string>
    </attribute>
    <widget class="QTreeWidget" name="treeFrameSets">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is a list of all the images contained in your game. 
Double-click on the one you want to edit.</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QToolButton" name="btnDeleteFrameSet">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Delete Image</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAddFrameSet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>25</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Add New Image</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <attribute name="title">
     <string>Levels</string>
    </attribute>
    <widget class="QTreeWidget" name="treeLevels">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is a list of all the levels contained in your game. 
Double-click on the one you want to edit.</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QToolButton" name="btnDeleteLevel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>20</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Delete Level</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAddLevel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>25</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Add New Level</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="lgck-builder.qrc">
       <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <attribute name="title">
     <string>Events</string>
    </attribute>
    <widget class="QTreeWidget" name="treeEvents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>181</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>This is a list of all the events for your game. 
Double-click on the one you want to edit.</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnClearAll">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="lgck-builder.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgGame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgGame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
